examples/Quick_Starts/XV15/submit_case_from_id.py

```diff
@@ -0,0 +1,21 @@
+"""Case submission from mesh ID's present on the Flexcompute servers"""
+
+import flow360 as fl
+
+case_name_list = []
+
+MESH_ID = "a823b33f-7523-4aa4-9881-99964917cc47"
+
+# submit case
+
+params = fl.Flow360Params("Flow360.json")
+case = fl.Case.create("XV15_Quick_Start", params, MESH_ID)
```

Comment:
@maciej-flexcompute Are we planning to support submit case from id by the end of Jan? Or this feature will be included later on?